libp2p/builders.nim

```diff
@@ -315,10 +314,13 @@ proc build*(b: SwitchBuilder): Switch {.raises: [LPError], public.} =
     ms = MultistreamSelect.new()
     muxedUpgrade = MuxedUpgrade.new(b.muxers, secureManagerInstances, ms)
 
+  if not b.autotls.isNil():
+    b.services.insert(b.autotls, 0)
```

Comment:
Why insert it into 0 pos? (curious because the wilcard service is inserted there too and that's what's used to resolve 0.0.0.0 IPs)

Reply:
It's just inserting into the very beginning of the sequence. It could be in the end too, it shouldn't make any difference.